src/main/java/io/jenkins/plugins/coverage/CoverageProcessor.java

```diff
@@ -388,19 +387,21 @@ private CoverageResult aggregatedResults(Map<CoverageReportAdapter, List<Coverag
     }
 
     /**
-     * Enable report auto-detect
+     * Setter for property 'reportDetector'.
      *
-     * @param autoDetectPath Ant-Style path to specify coverage report
+     * @param reportDetector report detector
      */
-    public void setAutoDetectPath(String autoDetectPath) {
-        this.autoDetectPath = autoDetectPath;
+    public void setReportDetector(Detector reportDetector) {
+        this.reportDetector = reportDetector;
     }
 
     /**
-     * @return <code>true</code> if auto detect is enable
+     * Getter for property 'reportDetector'.
```

Comment:
This comment does not hell much. Maybe it makes sense to just remove it
In such cases it's also better to use field links like `{@link #reportDetector}` so that field renaming gets applied in comments as well